libcloud/compute/types.py

```diff
@@ -207,25 +207,19 @@ class NodeState(object):
     :cvar PAUSED: Node is paused.
     :cvar UNKNOWN: Node state is unknown.
     """
-    RUNNING = 0
-    REBOOTING = 1
-    TERMINATED = 2
-    PENDING = 3
-    UNKNOWN = 4
-    STOPPED = 5
-    SUSPENDED = 6
-    ERROR = 7
-    PAUSED = 8
+    RUNNING = "running"
+    REBOOTING = "rebooting"
+    TERMINATED = "terminated"
+    PENDING = "pending"
+    UNKNOWN = "unknown"
+    STOPPED = "stopped"
+    SUSPENDED = "suspended"
+    ERROR = "error"
+    PAUSED = "paused"
 
     @classmethod
     def tostring(cls, value):
-        values = cls.__dict__
-        values = dict([(key, string) for key, string in values.items() if
-                       not key.startswith('__')])
-
-        for item_key, item_value in values.items():
-            if value == item_value:
-                return item_key
+        return value
 
     @classmethod
     def fromstring(cls, value):
```

Comment:
Yes, please leave this in.  We have a lot of code that uses these method to convert to/from the libcloud expected values, and removing this will break us.